schema/plugin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "jupyter.lab.setting-icon-class": "jp-FileIcon",
+  "jupyter.lab.setting-icon-label": "LaTeX",
+  "title": "LaTeX",
+  "description": "LaTeX settings.",
+  "properties": {
```

Comment:
is this how settings are described?

Reply:
Yes, they are described using [JSON-schema](http://json-schema.org/)

Reply:
i meant… i just want to be sure that I understand… automatically creates the Jlab specific meaning of "settings" as in configuration values that can be seen and set from within the "settings" ui in Jupyterlab

Reply:
Yes, that's right.

This, and the associated changes in the `package.json`, which allows them to be picked up and installed in the right spot.